library/src/f3d_loader_impl.cxx

```diff
@@ -320,13 +318,13 @@ bool loader_impl::loadFile(loader::LoadFileEnum load)
   }
 
   // Recover the importer
-  this->Internals->Importer =
-    ::GetImporter(filePath, this->Internals->Options.getAsBool("geometry-only"));
+  this->Internals->Importer = loader_impl::F3DInternals::GetImporter(
```

Comment:
this `F3D` prefix looks weird now. Can we use just `internals` instead of `F3DInternals` for pimpls?